controllers/comment.controller.js

```diff
@@ -14,4 +14,27 @@ exports.getCommentsByArticleId = (req, res, next) => {
 			res.status(200).send({ comments });
 		})
 		.catch(next)
-}
\ No newline at end of file
+}
+exports.postCommentByArticleId = (req, res, next) => {
+	const { article_id } = req.params;
+	const { username, body } = req.body;
+
+	if (!username || !body) {
+		return res.status(400).send({ message: 'Bad Request' })
+	}
+
+	selectArticleById(article_id)
+		.then((article) => {
+			if (!article) {
+				req.status(404).send({ message: err.message });
```

Comment:
This IF statement can never be true because your Promise.reject in the selectArticleByID function will take you straight to the catch block if the ID doesn't exist.

Reply:
done